beets/ui/commands.py

```diff
@@ -1666,6 +1666,7 @@ def print_completion(*args):
     u'/usr/share/local/bash-completion/bash_completion',
```

Comment:
Yah, seemed a bit strange to me... It's been around since 74cb897f79ff19a43eb100de16b700c1004067a3. I'll defer to the maintainers on making any changes. 😄